src/update.c

```diff
@@ -479,8 +467,9 @@ static int main_update()
 		(void)update_device_latest_version(server_version);
 		printf("Update was applied.\n");
 	} else if (ret < 0) {
-		// Ensure a positive exit status for the main program.
+		/* Ensure a positive exit status for the main program. */
```

Comment:
any reason to change the comment style? afaik we're using both in swupd

Reply:
For some reason I thought we were only using this style /* */, but now that you mentioned this I just started looking the other style of comments everywhere too. My mind was playing tricks on me. I'll return that comment the way it was.